src/main/java/racing/Car.java

```diff
@@ -0,0 +1,54 @@
+package racing;
+
+import com.sun.nio.sctp.IllegalReceiveException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Car {
+    private static final int MIN_NUMBER_OF_MOVES = 1;
+    private final List<Boolean> moves;
```

Comment:
공백 라인을 추가하면 상수와 인스턴스 변수를 직관적으로 구별하기 한결 쉬워질 것 같습니다.
```suggestion
    private static final int MIN_NUMBER_OF_MOVES = 1;
    
    private final List<Boolean> moves;
```